### DISCUSSION TITLE
Able to display all the log interface for executing SQL

### DISCUSSION THREAD

**Discussion by tianmingxing** [dbeaver/dbeaver#12401] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
I want to clearly know every command sent to the server for execution through the current client

**Describe the solution you'd like**
I need to confirm that my actions are as expected

**Describe alternatives you've considered**
HeidiSQL provides this very good function, it will be the effect I want.

**Additional context**
I want the feature snapshot screenshot, it comes from HeidiSQL software.
![image](https://user-images.githubusercontent.com/5615719/117090482-00c67e80-ad8b-11eb-86ae-b4c90c7532f9.png)

**Comment by p91paul** [comment, marked as answer]
go to view -> query manager. There are some options in preferences to increase its verbosity and write to a log file as well
